assets/js/romo/selected_options_list.js

```diff
@@ -47,87 +47,84 @@ RomoSelectedOptionsList.prototype.doRemoveItem = function(itemValue) {
 
 RomoSelectedOptionsList.prototype.doRefreshUI = function() {
   var itemValues  = this._getItemValues();
-  var uiListElem  = this.elem.find('.romo-selected-options-list-items');
-  var uiItemElems = uiListElem.find('.romo-selected-options-list-item');
-  var uiValues    = uiItemElems.get().map(function(uiItemNode) {
-    return $(uiItemNode).data('romo-selected-options-list-value');
-  });
-  var rmNodes = uiItemElems.get().filter(function(uiItemNode) {
-     return itemValues.indexOf($(uiItemNode).data('romo-selected-options-list-value')) === -1;
+  var uiListElem  = Romo.find(this.elem, '.romo-selected-options-list-items')[0];
+  var uiItemElems = Romo.find(uiListElem, '.romo-selected-options-list-item');
+  var uiValues    = uiItemElems.reduce(Romo.proxy(function(values, uiItemNodeElem) {
+    return values.concat([Romo.data(uiItemNodeElem, 'romo-selected-options-list-value')]);
+  }, this), []);
+  var rmNodeElems = uiItemElems.filter(function(uiItemNodeElem) {
```

Comment:
@jcredding should just call this `rmElems` now.  "Node" is kinda dead now that we aren't using jQuery (I previously used it to mean the raw un-wrapped elem).

Reply:
@jcredding also switch to `uiItemElem` (remove "Node" from all var names).